tests/TestHarness/TestHelper.py

```diff
@@ -199,3 +202,6 @@ def reportProductionAnalysis(thresholdMs):
                 Utils.Print("Cleanup wallet data.")
                 walletMgr.cleanup()
 
+        if trxGenLauncher is not None:
+            Utils.Print("Shut down the TransactionGeneratorsLauncher and TransactionGenerators")
+            trxGenLauncher.killAll()
```

Comment:
Is this still necessary, now that `trxGenLauncher` is being put on the `cluster` instance, and `cluster.killAll` will clean up the `self.trxGenLauncher`?